app/src/main/java/com/inveitix/android/clue/database/DBHelper.java

```diff
@@ -197,37 +233,61 @@ public void close() {
         database.close();
     }
 
+    /**
+     * Method for reading data from shapes table
+     * @return Cursor
+     */
     public Cursor getShapeValues() {
         return this.database.query(DBConstants.DB_TABLE_SHAPE, new String[]{
                         DBConstants.KEY_X, DBConstants.KEY_Y, DBConstants.KEY_ROOM_ID, DBConstants.KEY_MAP_ID, DBConstants.KEY_ID},
                 null, null, null, null, null);
     }
 
+    /**
+     * Method for reading data from doors table
+     * @return Cursor
+     */
     public Cursor getDoorValues() {
         return this.database.query(DBConstants.DB_TABLE_DOORS, new String[]{DBConstants.KEY_CONNECTED_TO,
                         DBConstants.KEY_MAP_ID, DBConstants.KEY_ROOM_ID, DBConstants.KEY_ID, DBConstants.KEY_X, DBConstants.KEY_Y},
                 null, null, null, null, null);
     }
 
+    /**
+     * Method for reading data from QRs table
```

Comment:
Repeats what the name of the method says. Either elaborate or remove documentation.